webroot/src/components/Licensee/LicenseeList/LicenseeList.ts

```diff
@@ -169,8 +184,10 @@ class LicenseeList extends Vue {
             }
         });
 
-        // Temp for limited server paging support
-        if (this.licenseStore.error?.message === 'end of list' && page > 1) {
+        // Support for limited server paging support
+        if (this.licenseStore.error?.message instanceof PageExhaustError) {
+            this.prevKey = '';
+            this.nextKey = '';
             await this.setDefaultPaging(true);
             await this.fetchListData();
         }
```

Comment:
My read of this is: If we hit the end of the list, go back to the beginning. Why not just stay where we already are?

Reply:
Done.

Reply:
Wouldn't we want to keep the keys (at least the prevKey) the same, so proper pagination is still supported?

Reply:
Ah gotcha. I updated so it's just a full no-op.

Reply:
Sounds good! (To clarify, I'm not committed to this route- more just noting an assumption I'd make without further documentation/explanation. If you prefer the prior approach, that's good with me, as long as there's a brief comment explaining its purpose.)